### ISSUE TITLE
DM: Oauth2 session UI

### ISSUE THREAD

**Issue by matrixbot** [element-hq/matrix-authentication-service#1499]
*This issue was originally created by [**@kerryarchibald**](https://github.com/kerryarchibald) at <https://github.com/matrix-org/matrix-authentication-service/issues/1499>.*

Depends on #1498 

![](https://hackmd.io/_uploads/BJv0N5gh2.png)
![](https://hackmd.io/_uploads/Syl4B9x2n.png)

`/account/oauth2-sessions` route better matches designs

Note: designs do not reflect the data available, some information will not be included. Also, as the current API does not support aggregation by client, the client information should be included on the tile.

Out of scope:
- IP
- Rename
- Push notifs

AC:
- Session name inferred from deviceId in scope, else session`id`
- Sign out button kills the session
- Signed out sessions are removed from the list when 'active only' filter is enabled
- Finished sessions display 'finished at' under createdAt
- Finished sessions do not have a sign out button
- Client name is displayed on the tile
- Client icon is displayed on the tile when present
- Clicking the session name links to session detail route

Displaying the device type icon using `Oauth2Client.applicationType` is split off into https://github.com/matrix-org/matrix-authentication-service/issues/1551.
